modules/web-console/src/main/java/org/apache/ignite/console/WebConsoleServer.java

```diff
@@ -105,23 +114,48 @@ protected void logError(String msg, Throwable e) {
     }
 
     /**
+     * @param cfg Configuration.
      * @param ignite Ignite.
      * @param auth Auth provider.
      * @param cfgsRouter Configurations REST API router.
      * @param notebooksRouter Notebooks REST API router.
      */
     public WebConsoleServer(
+        WebConsoleConfiguration cfg,
         Ignite ignite,
         IgniteAuth auth,
         RestApiRouter cfgsRouter,
         RestApiRouter notebooksRouter
     ) {
+        this.cfg = cfg;
         this.ignite = ignite;
         this.auth = auth;
         this.cfgsRouter = cfgsRouter;
         this.notebooksRouter = notebooksRouter;
     }
 
+    /**
+     * @param path Path to JKS file.
+     * @param pwd Optional password.
+     * @return Java key store options or {@code null}.
+     */
+    @Nullable private JksOptions jksOptions(String path, String pwd) {
```

Comment:
Why you not use same method from AgentUtils

Reply:
I do not want to have dependency between Agent and Web Console Backend.